environment.yml

```diff
@@ -2,20 +2,14 @@ name: jump_sc
 channels:
   - conda-forge
 dependencies:
-  - conda-forge::python
-  - conda-forge::boto3
-  - conda-forge::jupyter
-  - conda-forge::jupyterlab
-  - conda-forge::ipykernel
-  - conda-forge::nb_conda_kernels
-  - conda-forge::jupyter_contrib_nbextensions
-  - conda-forge::pandas
-  - conda-forge::scipy
-  - conda-forge::numpy
-  - conda-forge::scikit-learn
-  - conda-forge::plotnine
-  - conda-forge::umap-learn
-  - conda-forge::matplotlib
+  - conda-forge::python=3.9
+  - pip
   - pip:
-    - git+https://github.com/d33bs/cytotable@c6e0046e76986f9aa837b441186b621fe06b8189
-    - duckdb==0.8.0
+    - pathlib2
+    - numpy
+    - pandas
+    - matplotlib
+    - git+https://github.com/cytomining/pycytominer.git@8e3c28d3b81efd2c241d4c792edfefaa46698115
+    - scipy==1.8.1
+    - scikit-learn
```

Comment:
Many of these packages are available on conda-forge (pathlib, numpy, pandas, matplotlib, scipy, scikit-learn). The only one to download this way is pycytominer.

See the change log and make sure you install the primary python packages through conda-forge.

It is good practice to limit pip usage in a conda environment: https://www.anaconda.com/blog/using-pip-in-a-conda-environment

Reply:
That's good to know, the conda environment file should be updated now.